public class ExceptionEx5{
	public static void main(String[] ar){
		// Exception을 강제로 발생시킨다.
		throw new ArrayIndexOutOfBoundsException();
	} 
}